src/containers/NewProject.js

```diff
@@ -316,7 +316,7 @@ class NewProject extends Component {
       license: this.state.dropDownText,
       author: this.state.author,
       version: this.state.version,
-      sourceControl: sourceControlId,
+      sourcecontrol: sourceControlId,
```

Comment:
Tyr takes a param called `sourceControl` (According to the README). We should stick with the same names across tyr, endor, and yggdrasil